documentation/CONFIG.md

```diff
@@ -13,9 +13,14 @@ Connector-specific configuration properties are described below.
 
 ``scylladb.contact.points``
 
-  The ScyllaDB hosts to connect to. Scylla nodes use this list of hosts to find each other and learn the topology of the ring. You must change this if you are running multiple nodes.
-  It's essential to put at least 2 hosts in case of bigger cluster, since if first host is down, it will contact second one and get the state of the cluster from it.
-  Eg. When using the docker image, connect to the host it uses.
+  The ScyllaDB hosts to connect to. Scylla nodes use this list of hosts to find each other and learn the topology of the ring. 
+  You must change this if you are running multiple nodes.
+  It's essential to put at least 2 hosts in case of bigger cluster, since if first host is down, 
+  it will contact second one and get the state of the cluster from it.
+  Eg. When using the docker image, connect to the host it uses. 
+  To connect to private Scylla nodes, provide a JSON string having all internal private network address:port mapped to
+  an external network address:port as key value pairs. 
+  Eg. {\"10.0.24.69:9042\": \"sl-eu-lon-2-portal.3.dblayer.com:15227\", \"10.0.24.71:9042\": \"sl-eu-lon-2-portal.2.dblayer.com:15229\", \"10.0.24.70:9042\": \"sl-eu-lon-2-portal.1.dblayer.com:15228\"}
```

Comment:
Can you explain this a bit more... @patitapaban19 - I find this a bit confusing...